docs/books/ownCloud_iOS_App_Manual.adoc

```diff
@@ -8,5 +8,4 @@ The ownCloud Team <docs@owncloud.com>
 :toclevels: 1
 :icons: font
 :icon-set: octicon
-
-include::modules/ROOT/pages/index.adoc[leveloffset=+1]
+:module_base_path: modules/ROOT/pages/
```

Comment:
That should not be required?

Reply:
It has now been removed. Can you review again, pls?